pkg/planner/core/stats.go

```diff
@@ -532,16 +531,6 @@ func initStats(ds *logicalop.DataSource) {
 		if col.State != model.StatePublic {
 			continue
 		}
-		// If we enable lite stats init or we just found out the meta info of the column is missed, we need to register columns for async load.
```

Comment:
meta is missed=> need load
meta is there while lite init will miss loading hist => need load as well